### DISCUSSION TITLE
Cannot resolve a specific FQND

### DISCUSSION THREAD

**Discussion by overrunner** [AdguardTeam/AdGuardHome#2738] [Q&A]
- [X] I am running the latest version
- [X] I checked the documentation and found no answer
- [X] I checked to make sure that this issue has not already been filed

### Issue Details

I cannot resolve the URL: www.home24.de -> home24.de does work, but when I try to resolve www.home24.de I get errors:

---- ASKING ADGUARD -----
dig www.home24.de @1.1.1.1

; <<>> DiG 9.10.6 <<>> www.home24.de @1.1.1.1
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: NOERROR, id: 56132
;; flags: qr rd ra; QUERY: 1, ANSWER: 3, AUTHORITY: 0, ADDITIONAL: 1

;; OPT PSEUDOSECTION:
; EDNS: version: 0, flags:; udp: 1232
;; QUESTION SECTION:
;www.home24.de.			IN	A

;; ANSWER SECTION:
www.home24.de.		300	IN	CNAME	www.home24.de.san.edgekey.net.
www.home24.de.san.edgekey.net. 21600 IN	CNAME	e3502.b.akamaiedge.net.
e3502.b.akamaiedge.net.	20	IN	A	104.103.108.205

;; Query time: 2211 msec
;; SERVER: 1.1.1.1#53(1.1.1.1)
;; WHEN: Sun Jan 31 23:07:56 CET 2021
;; MSG SIZE  rcvd: 134

---- END -----

----- ASKING 1.1.1.1 directly -----
* **Version of AdGuard Home server:**
  * 0.104.3
* **How did you install AdGuard Home:**
  * Docker
* **How did you setup DNS configuration:**
  * Upstream Server: 1.1.1.1
tested with DNSSEC on/off and with TLS on/off -> neither works
Even deactivated every Filter List.

### Expected Behavior
www.home24.de should be resolved properly

### Actual Behavior
No response. even no log entry.

**Comment by ameshkov** [comment]
Could it be that you have it in the disallowed domains list?

Anyways, logs should help:

1. [Configure](https://github.com/AdguardTeam/AdGuardHome/wiki/Configuration#configuration-file) AdGuard Home to collect logs:
    * Specify `log_file`
    * Set `verbose` to `True`
2. Restart AdGuard Home and reproduce the issue
3. Post the log file here.
